_adr/81/index.adoc

```diff
@@ -19,10 +19,267 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
+            "maxPartitions": "2000", "totalMaxConnections": "1000",
+            "maxConnectionAttemptsPerSec": "100", "maxMessageSize": "1",
+            "cost": 2,
+          }
+        },
+        { "id": "l",
+          "parameterValues": {
+            "ingressThroughputPerSec": "90", "egressThroughputPerSec": "90", 
+            "maxPartitions": "3000", "totalMaxConnections": "1500",
+            "maxConnectionAttemptsPerSec": "100", "maxMessageSize": "1".
+            "cost": 3,
+          }
+        }
+      ],
+    },
+    { "id": "eval", 
+      // ...
+    }
+    // other profiles as necessary
+]
+----
+<1> The order of this list MAY be reflect the way that profiles are shown in the UI. 
+E.g. "Developer", "Standard" and "Enterprise" profiles might be shown in that order.
+<2> Each profile's `id` uniquely identifies the profile. 
+It MUST NOT change.
+It should be treated as an internal id and MUST NOT be exposed directly to a user.
+<3> Each profile defines a set of parameters, together with their units and descriptions. 
+Each size in the profile must give values for all the parameters that the profile defines. 
+Different profiles MAY define different parameters. 
+The units and descriptions MAY be localized.
+<4> The size list MUST be ordered such that later items represent clusters with more capacity.
+<5> The size `id` uniquely identifies the size within the profile. 
+It MUST NOT change.
+It should be treated as an internal id and MUST NOT be exposed directly to a user.
+The unit or meaning of cost is not defined in this ADR. 
+<7> The `parameterValues` of a size MUST define the same properties as the profile's `parameters` object.
+
+NOTE: The above is not intended as a detailed API specification, merely an illustration of how the profiles and their sizes are communicated from the Fleet Manager to a UI.
+
+Because the profile and size ids never change a UI MAY use them as keys for localized user-facing names and descriptions.
+
+Where it is necessary for a single string to be used to unambiguously refer to a given size within a profile the format `<profileId>.<sizeId>` SHOULD be used. 
+To avoid ambiguity profile and size ids MUST be a valid DNS subdomain.
+
+It may be necessary, eventually, to distinguish the supported actions that are supported for a given profile and size.
+For example, we might want to prevent the creation of new instances of the "eval" profile while supporting existing instances. 
+So we might eventually want additional properties in the instance size schema.
+
+==== Instance creation
+
+The `POST /api/kafkas_mgmt/v1/kafkas` endpoint would change to take the profile id and size id in addition to the existing parameters.
+
+[source,json]
+.Example `POST /api/kafkas_mgmt/v1/kafkas` made by a UI to the Fleet Manager to create an instance
+----
+{
+    "region": "us-east-1",
+    "cloud_provider": "aws",
+    "profileId": "standard", <1>
+    "sizeId": "s", <2>
```

Comment:
@tombentley - just to confirm, based on the discussion held earlier today - UI will send one parameter (called `plan`), in format `<instance_type>.<size_id>`. is that correct?

Reply:
I think @pmuir was happy with either (as am I).